src/Dhall/Core.hs

```diff
@@ -473,6 +478,7 @@ instance Monad (Expr s) where
     Field a b            >>= k = Field (a >>= k) b
     Project a b          >>= k = Project (a >>= k) b
     Note a b             >>= k = Note a (b >>= k)
+    ImportAlt a _b _c    >>= k = k a  -- TODO: does this make sense?
```

Comment:
Yes, this makes sense

Reply:
The part I'm not sure about is that the contained expressions (`_b, _c`) will not bind to `k` (as it happens with the other operators